src/main/java/com/fidesmo/tutorials/hellofidesmo/MainActivity.java

```diff
@@ -100,42 +102,40 @@ protected void onNewIntent(Intent intent) {
 
     @Override
     public void tagDiscovered(Tag tag) {
-        Log.i(TAG, "Card detected on the NFC interface!");
+        setMainMessage("Reading card...");
```

Comment:
Why not use `R.string.reading_card` as before? Even for a short tutorial, it is still a good practice...